config/mycnf/default.cnf

```diff
@@ -14,9 +14,6 @@ port = {{.MysqlPort}}
 secure-file-priv = {{.SecureFilePriv}}
 {{end}}
 
-# all db instances should start in read-only mode - once the db is started and
-# fully functional, we'll push it into read-write mode
-read-only
```

Comment:
I thought we agreed that we still need to keep this because of MariaDB. Other option is to add these lines to mariadb10.cnf.